api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -122,6 +122,49 @@
    */
   void setAttribute(String key, boolean value);
 
+  /**
+   * Sets an attribute to the {@code Span}. If the {@code Span} previously contained a mapping for
+   * the key, the old value is replaced by the specified value.
+   *
+   * <p>If a null or empty String {@code value} is passed in, the attribute will be silently
+   * dropped. Note: this behavior could change in the future.
+   *
+   * @param key the key for this attribute.
+   * @param values the value for this attribute.
+   * @since 0.1.0
+   */
+  void setAttribute(String key, String... values);
```

Comment:
I wouldn't add this to the Span Api to keep the surface low. I would rely on the version with AttributeValue for this usecase.

Reply:
I agree with this. Let's try to keep the `Span` surface whenever possible.